### PR TITLE
Creation of an nmr-preprocessing GTM

### PR DESCRIPTION
**Pull request by mtremblayfr**
# Thank you for contributing to the GTN! :yellow_heart:

## Please check that:

- [ ] Your pull request has a **good title**, e.g.
    - "Fix typo in ansible-galaxy tutorial"
    - "Add new transcriptomics tutorial covering a new sequencing technology"
- [ ] Please **replace this entire pull request message** with a *description of your changes*, it will help us review your pull request faster
- [ ] Check that your images are allowed to be re-hosted by the GTN

Once these are done, you're ready to go!

<details>
    <summary>Consider sustainable computing with Draft Mode :green_heart:</summary>

Is your pull request finished? 100% ready to be merged? Or do you want some time to work on it first (which we encourage! It's great to have visibility)

If not, then please consider creating this pull request as a draft.

Regular pull requests will trigger automated test runs when they are created
and every time you update them.

As a small contribution to sustainable computing, we skip the most energy intensive of these tests for pull requests marked as drafts. Once your pull request is ready to go, then you can click a button saying "Ready for Review" and the tests will be run!
</details>

### CODE REVIEW

**Comment by mtremblayfr** [comment]
Work in progress

**Comment by hexylena** [comment]
Hi @mtremblayfr great to see this work in progress! I've moved it to 'draft' mode now, please click "ready for review" when you're ready :)

File: topics/metabolomics/tutorials/nmr-preprocessing/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,1023 @@
...
+- To get familiar with the use of PEPS-NMR-based Galaxy modules dedicated to 1H-NMR data preprocessing.
+- To evaluate the potential of a workflow approach when dealing with 1H-NMR data preprocessing.
+time_estimation: 2H
+key_points:
+- To process untargeted 1H-NMR based metabolomic data preprocessing, various steps are needed.	
+- Resources are available in Galaxy, but do not forget that you need appropriate knowledge to perform a relevant preprocessing.
+contributors:
+- mtremblayfr
+- Cécile Canlet
+- ManonMartin
```
Comment (hexylena) [inline topics/metabolomics/tutorials/nmr-preprocessing/tutorial.md:20]: If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

**Comment by mtremblayfr** [comment]
Work in progress. Does somebody accept it to finish the GTN please?

**Pull request closed without merging**